inference/mlflow-huggingface/src/models/bert-base-uncased/model.yaml

```diff
@@ -1,11 +1,8 @@
 path:                                                                 #Required
-  # local: "./local-path-to-model-files"                              #Required  Either remote or local should be passed. if both are given, local is given preference  
-  remote:                                                             #Required. Either remote or local should be passed. if both are given, local is given preference  
-    type: git                                                         #Required. Download Type. Can be git, http or ftp
-    uri: https://huggingface.co/bert-base-uncased                     #Required. Download URI
-    commit_hash: 5546055f03398095e385d7dc625e636cc8910bf2             #Optional
+  type: git                                                           #Required
```

Comment:
i thought you guys said EV2 doesn't support this?

Reply:
We can still support git for smaller models.

Reply:
Is `git` installed available on Ev2 agents?

Reply:
we can install git in the requirements if not already available. Similar to `az-copy`

Reply:
Resolving this. Current models are to be downloaded from blobstore